atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/events/Event.java

```diff
@@ -48,6 +50,7 @@ static Event fromKeywordMap(Map<Keyword, ?> map) {
     static Map<Keyword, Object> toKeywordMap(Event event) {
         Map<String, Object> rawStringMap = OBJECT_MAPPER.convertValue(event, new TypeReference<Map<String, ?>>() {});
         return EntryStream.of(rawStringMap)
+                .filterValues(Objects::nonNull)
```

Comment:
Yep, as you suggest this isn't an issue because you'll get an `IllegalStateException` when trying to build the event without a required property.

I'll add the test that verifies that this can't be done.